.prettierrc.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
   requirePragma: false,
   singleQuote: true,
   tabWidth: 2,
-  trailingComma: 'none',
+  trailingComma: 'es5',
   useTabs: false,
-  printWidth: 80
-};
+  printWidth: 80,
+  semi: false,
```

Comment:
Enabling `"semi"` as `false` reduced the ESLint from 601 to 283 😐

Reply:
that is a significant difference! have we established a code pattern for using vs not using semis across projects? i am indifferent either way.

Reply:
I'm indifferent as well, as long as we have this configured on prettier I'm good 😂

Reply:
I personally choose semis because it's closer to other languages / has more readability (don't have to think about when statements end), but I'm mostly indifferent because I just have vscode configured to lint automatically based on the config 😁